securesystemslib/rsa_keys.py

```diff
@@ -381,7 +437,8 @@ def create_rsa_signature(private_key, data, scheme='rsassa-pss-sha256'):
 
 
 
-def verify_rsa_signature(signature, signature_scheme, public_key, data):
+def verify_rsa_signature(signature, signature_scheme, public_key, data,
+  salt_length_type=HashSaltLengthType):
```

Comment:
Same as above, we usually just call `securesystemslib.keys.verify_signature` and use the key to dispatch to the algorithm specific function.